draft-ietf-jsonpath-iregexp.md

```diff
@@ -92,7 +92,8 @@ in this specification; the plural is "I-Regexps".
 {::boilerplate bcp14-tagged}
 
 The grammatical rules in this document are to be interpreted as ABNF,
-as described in {{-abnf}} and {{-abnf-cs}}.
+as described in {{-abnf}} and {{-abnf-cs}}, where the "characters" of
+{{Section 2.3 of -abnf}} are Unicode scalar values.
```

Comment:
Is Section 2.3 right?  Maybe I followed it wrong, but I get https://www.rfc-editor.org/rfc/rfc5234#section-2.3, which is titled "Termination Values."

Reply:
It's "Terminal Values", and the first paragraph explains that these used to be ASCII characters, and the new text explains that the terminal values are Unicode scalar values.

Reply:
okay.  (not sure why I can't mark this resolved)